examples/slo/main.tf

```diff
@@ -58,6 +58,27 @@ resource "chronosphere_slo" "slo_with_signal_grouping_signal_per_series" {
     reporting_windows {
       duration = "28d"
     }
+    burn_rate_alerting_config {
```

Comment:
related to my previous comment.. if i reorder these it should still result in no diff in `terraform plan` but i have a feeling it would.

Reply:
They are equivalent configurations, and the UI will show it in the order that the burn rates were defined -- for example user may change it to have CRITICAL before WARN conditions or vice versa. I feel it's ok to have a reorder cause a diff.